docs/topics/install/docker.rst

```diff
@@ -140,6 +135,81 @@ migrations::
 Gotchas!
 ~~~~~~~~
 
+Here's a list of a few of the issues you might face when using docker.
+
+Can't access the web server?
+----------------------------
+
+Check you've created an hosts file entry pointing ``olympia.dev`` to the
+relevant ip address.
+
+Also make sure you've copied ``docker-compose.override.yml.tmpl`` to
```

Comment:
Ok, there is another way to do this. It will make it more incovenient if you want to change the ports but will mean the default requires no extra changes. I'll update the docs to reflect that way since I think it will be nice to not require a breaking change.